src/plugin_config.cpp

```diff
@@ -50,11 +50,12 @@ void from_json(const nlohmann::json& j, PluginConfig& cfg) {
         }
     }
     if (cfg.engines.cri.sockets.empty()) {
+        cfg.engines.cri.sockets.emplace_back("/run/containerd/containerd.sock");
```

Comment:
Same defaults we have in libs master: https://github.com/falcosecurity/libs/blob/release/0.20.x/userspace/libsinsp/container.cpp#L578